ls1intum/Artemis
# [Feature] Shorten student's repository link

### Checklist

### Motivation and Context


If you open a student's repository from within Artemis (e.g. from the Exercise Scores View, Exercise Participation View, Exam Summaries View, etc.), the student's LRZ-ID is added as a user name in the URL. Due to that, one gets a pop-up asking to confirm if you want to log in to the website using the student's ID instead of just opening the repository straight away.
Fixes: #2970 

### Description

The repository url of a student gets cropped by its preceding lrz-id. Everywhere where a tutor has the possibility to access the repository via link.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create a programming exercise and an exam and participate with a student
4. Now as a tutor click within e.g. the exam summary or exercise score page on the repository link. It should open the requested repository without question. (Actually this is just an issue on the production environment)

### Test Coverage



No changes.

### Screenshots


#### Wrong students repo url - It should not show: 
![image](https://user-images.githubusercontent.com/33484318/110157852-f9194680-7de8-11eb-9af9-7ea3e8ca62dc.png)

#### Correct students repo url - It should show: 
![image](https://user-images.githubusercontent.com/33484318/110160576-7c886700-7dec-11eb-8a25-c5ef2d760cc3.png)
